support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -45,6 +45,24 @@ export const pageUrlRegexes = {
 };
 
 export const tests: Tests = {
+	patronsOneOffOnly: {
+		variants: [
+			// not really an AB test
+			{
+				id: 'variant',
+			},
+		],
+		audiences: {
+			ALL: {
+				offset: 0,
+				size: 0,
+			},
+		},
+		isActive: true,
+		referrerControlled: true,
```

Comment:
this means users will only go into this test if it's defined in the url querystring